### DISCUSSION TITLE
LFX Mentorship '23 Term 01 March - May

### DISCUSSION THREAD

**Discussion by nate-double-u** [cncf/mentoring#763] [Announcements]
Hello everyone!

The LFX Mentorship 2023 Term 01 March - May is open now open for project ideas:
https://github.com/cncf/mentoring/tree/main/programs/lfx-mentorship/2023/01-Mar-May

Project submission and application timeline:
* project proposal period: January 16 - 31
* mentee applications open: February 1 - 21 (2 weeks)
* application review/admission decisions/HR paperwork: February 22 - March 7

We're looking forward to seeing all the project ideas you're interested in working on!

## Timeline

Mentorship duration - three months (12 weeks - full-time schedule)

| activity | date |
| --- | --- |   
| project proposals | January 16 - 31, 5:00 PM PDT |
| mentee applications open | February 1 - 21, 5:00 PM PDT |
| application review/admission decisions/HR paperwork | February 22 - March 7, 5:00 PM PDT |
| Mentorship program begins with the initial work assignments | March 8 (Week 1) | 
| Midterm mentee evaluations and first stipend payments | April 12 (Week 6) |
| Final mentee evaluations and mentee feedback/blog submission due, second and final stipend payment approvals | May 24, 5:00 PM PST (Week 12) |
| Last day of term | May 31 |

[original announcement email](https://lists.cncf.io/g/tag-cs-mentoring-wg/message/7)

**Comment by nate-double-u** [comment]
A friendly reminder, the deadline for project proposals is about a week away (Jan 31). So please get your project ideas in!

https://github.com/cncf/mentoring/tree/main/programs/lfx-mentorship/2023/01-Mar-May

**Reply by HEEKEE002** [reply]
Hey @andreuxxxx i see that the application are closed, due to some issue, I didn't applied recently, but i see that application are now closed, please is there any way to apply,

I want to learn many many new things from this mentorship, I would be grateful to you, if you somehow can help,. I really need this mentorship

**Reply by nate-double-u** [reply]
Hi @HEEKEE002, applications were closed on Feb 21, 2023 (after having been extended for a week). I'm afraid that we can't reopen applications at this point as the mentors are already going through and making decisions.

However, other programs are running. Take a look at [GSoC 2023](https://github.com/cncf/mentoring/discussions/848), which is still in the planning and contributor proposal phase. You can also take a look at the next term of LFX mentorship, we have published the timelines for both [Term 02-Jun-Aug](https://github.com/cncf/mentoring/tree/main/lfx-mentorship/2023/02-Jun-Aug), and [Term 03-Sep-Nov](https://github.com/cncf/mentoring/tree/main/lfx-mentorship/2023/03-Sep-Nov).

**Comment by nate-double-u** [comment]
We are having a technical issue with the LTX Mentorship platform which is preventing us from opening new projects.

The team is aware of the issue and is working to correct it, but this work may take a couple of days.

When I know a estimated time to fix it I will share it here. If this takes more than a day or so to fix we will extend the application window and other deadlines as needed.

Thanks for your patience, we'll get this fixed as soon as possible.

**Reply by nate-double-u** [reply]
Update: we are still having a technical issue preventing us from opening projects for application. When we do open applications we will adjust the dates to make sure folks still have time to apply and assess applications.

Thanks for your continued patience, we'll get this fixed as soon as possible.

**Comment by feelinspired** [comment]
This issue has been fixed and all the new CNCF Programs have now been published on the LFX Mentorship Platform.

**Reply by nate-double-u** [reply]
Thanks for the help on this @feelinspired!

You can find all the accepted projects on this page: https://github.com/cncf/mentoring/tree/main/lfx-mentorship/2023/01-Mar-May

There are a couple project that will be coming in the next day or so, but please feel free to start applying. I will also update the timeline to reflect this delay.

**Comment by nate-double-u** [comment]
Hello Everyone, 

All projects currently accepting applications can be found here: https://github.com/cncf/mentoring/tree/main/lfx-mentorship/2023/01-Mar-May

Please note: With the updated timeline, we've pushed all the due dates back one week to compensate for the time lost due to the delay in getting projects approved.

**Comment by nate-double-u** [comment]
Reminder - mentee applications are due Feb 21!  This is a great opportunity to get started in the cloud native community!

https://github.com/cncf/mentoring/tree/main/lfx-mentorship/2023/01-Mar-May

**Comment by nate-double-u** [comment]
Don't forget ⏱️ Mentee applications for the #LFX Program (March-May) close Feb 21!
We have more than 60 opportunities from projects including #Cilium, #Kubernetes, #Linkerd + MANY more!
Learn more about the projects and apply today: https://github.com/cncf/mentoring/tree/main/lfx-mentorship/2023/01-Mar-May

**Comment by nate-double-u** [comment]
Thanks for applying everyone!

The LFX Platform has been closed for applications, and the mentors are now reviewing all of the submissions. The application review/admission decisions/HR paperwork phase will end on March 7.

**Comment by nate-double-u** [comment]
Hello Everyone,

Due to the volume of project proposals and applicants (thanks everyone, this is actually a nice problem to have), I'm not going to be able to do all of the data entry on the final due date like I normally do.

I'm going to process applicant selections on a first come first serve basis -- currently there aren't any conflicts (the same mentee selected for one or more projects). 

This means that some applicants will be notified before others, but I'll have them all done by the deadline of March 7th so that the term can start on March 8th.

Accepted mentees will be notified before declined applicants because of how the platform works.

**Comment by Chinwendu20** [comment]
Hello please does this mean the earlier one applied the better their chances of getting selected as a mentee? Thanks for all the good work.

**Reply by nate-double-u** [reply]
No, _all_ applicants are being considered. This is about the mentor's making a selection.

**Comment by nate-double-u** [comment]
Today is the first day of the 01 Mar-May semester! Congratulations to all those who were selected as mentees 🎉

We are still working on getting offer letters out to those who were selected, as well as notifications to those who were not selected. Thank you all for your patience.

If you weren't selected, please don't be discouraged! We run three terms a year for the CNCF LFX program, as well as participate in GSoC and GSoD, there are still plenty of opportunities to participate.

And lastly, a big thanks to all the project maintainers and contributors who wrote up proposals and who are spending their time as mentors this term!

**Reply by meobilivang** [reply]
Hi Nate @nate-double-u! Excited to be a part of this fantastic program  🥂 🎉 🎉  

Before moving forward, I am working on the paperwork and have been trying to contact someone at Linux Foundation for support. I would be so grateful if you can share a point of contact of whom I can speak to.

Thank you so much 🙏 Have a great day, sir 🙇

**Reply by nate-double-u** [reply]
Hi @meobilivang, I've received your email. I'll respond there as there are some confidential bits to discuss.

**Reply by Chinwendu20** [reply]
Hi, I would also like some clarification concerning the paperwork. How do I send a mail as well

**Comment by xuyangwang0825** [comment]
Hi there,
May I ask if not receiving any notifications means that I'm not selected for the project? My status on the LFX portal still shows as pending.

**Reply by nate-double-u** [reply]
Hi @xuyangwang0825, I'm sorry for the delay -- both in my response here and for notifications being late. We're still working on an automated way of sending out decline notices. If you haven't received a notice that you have been accepted, you can assume that you haven't been. We'll work on doing this better in future, and I appreciate your patience as we change and update our systems and processes.
